### ISSUE TITLE
Create team overview pages

### ISSUE THREAD

**Issue by Nik-09** [oppia/oppia-web-developer-docs#28]
Add the getting-started pages for the following two pages of Oppia. The pages should contain similarly detailed information as we have for the [contributor dashboard](https://github.com/oppia/oppia/wiki/Contributor-dashboard).
- [x] LaCE @SanjaySajuJacob 
- [ ] Dev workflow

**Comment by seanlip** [comment]
Per discussion with @nithinrdy, maybe the correct entry point for this is through the per-team getting-started pages -- here's the one for [Contributor Dashboard](https://github.com/oppia/oppia/wiki/Contributor-dashboard), and we might want similar ones for the other two teams, LaCE and dev workflow.

**Comment by Shivkant-Chauhan** [comment]
@Nik-09 please update the project board with the expected merge date.

**Comment by Shivkant-Chauhan** [comment]
hi @Nik-09, just a follow-up for this issue, can you please update the expected merge date for this issue. thanks!

**Comment by SD-13** [comment]
@Nik-09 Are you still working on this issue? Please let me know so that I can update the project board.

**Comment by Nik-09** [comment]
> Per discussion with @nithinrdy, maybe the correct entry point for this is through the per-team getting-started pages -- here's the one for [Contributor Dashboard](https://github.com/oppia/oppia/wiki/Contributor-dashboard), and we might want similar ones for the other two teams, LaCE and dev workflow.

I have updated the description as per the comment.

**Comment by seanlip** [comment]
Hi @Nik-09 -- @SanjaySajuJacob has actually prepared this doc for LaCE onboarding: https://github.com/oppia/oppia/wiki/LaCE-onboarding-guide. Do you think it is sufficient for this issue?

Also @U8NWXD @gp201 @JeeveshGarg @Shivkant-Chauhan do you want to make a similar doc for folks onboarding to the dev workflow team? I'm not sure what should go on it, so checking to see if there are any common issues that arise for new contributors (or contributors starting a project in a new domain) that you're aware of. Possibly even a list of links to the relevant wiki pages might be helpful (e.g. "If you're working on XYZ, look at ABC").

**Comment by Nik-09** [comment]
> Hi @Nik-09 -- @SanjaySajuJacob has actually prepared this doc for LaCE onboarding: https://github.com/oppia/oppia/wiki/LaCE-onboarding-guide. Do you think it is sufficient for this issue?

Yes, [the page](https://github.com/oppia/oppia/wiki/LaCE-onboarding-guide) is very effective; I think adding an overview section for the LaCE team will be better.

**Comment by seanlip** [comment]
Thanks @Nik-09 for the suggestion! Added to #156.

Also @U8NWXD @gp201 @JeeveshGarg  @Shivkant-Chauhan, just a friendly ping on https://github.com/oppia/oppia-web-developer-docs/issues/28#issuecomment-1506600546 -- thanks!

**Comment by U8NWXD** [comment]
@seanlip I can't think of any common issues that we don't already have docs for. Right now we're spread over a pretty broad range of projects, so we might have more use for a centralized page once we're all working on more similar tasks

**Comment by seanlip** [comment]
@U8NWXD OK, but in that case, what do you think about a "Welcome to the Dev Workflow Team" page that lists the broad areas and points to the relevant wiki pages for each? Then we have a getting started page for each team, which we can perhaps include in the emails we send folks when inviting them to a team (or in the getting started page).

(Just double-checking this point since your comment didn't address it directly; if you still don't want it, then that's fine!)

**Comment by U8NWXD** [comment]
I think that could be useful. Here are the pages I can think of to link:
* [Automated testing](https://github.com/oppia/oppia/wiki/Tests) and [Debugging](https://github.com/oppia/oppia/wiki/Debugging) pages
* [Lint checks](https://github.com/oppia/oppia/wiki/Lint-Checks)
* [Backend typing](https://github.com/oppia/oppia/wiki/Backend-Type-Annotations)
* [Codebase overview](https://github.com/oppia/oppia/wiki/Overview-of-the-Oppia-codebase)
* [Oppiabot](https://github.com/oppia/oppia/wiki/Oppiabot)
* [Third-party libs](https://github.com/oppia/oppia/wiki/Third-party-libraries)
* [Build process](https://github.com/oppia/oppia/wiki/Build-process)
* [How the wiki works](https://github.com/oppia/oppia/wiki/Wiki)
